tests/subsys/net/lib/nrf_cloud/src/main.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ztest.h>
+#include <zephyr.h>
+#include <stdio.h>
+#include <net/nrf_cloud.h>
+#include "nrf_cloud_fsm.h"
+
+/* Main test function that declares a test suite
+*  includes all the test cases implemented below
+*/
+ZTEST_SUITE(nrf_cloud_init_test, NULL, NULL, NULL, NULL, NULL);
+
+/* The library can only be initialized when the internal state is
+*  idle, so the test verifies that nrf_loud_init function will
```

Comment:
```suggestion
*  idle, so the test verifies that nrf_cloud_init function will
```

Reply:
oops I missed that little boo boo there, thanks a ton